src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -192,21 +194,35 @@ class DeepDiveLogParser extends JavaTokenParsers {
     | lexpr
     )
 
-  def lexpr : Parser[Expr] =
-    ( functionName ~ "(" ~ rep1sep(expr, ",") ~ ")" ^^ {
-        case (name ~ _ ~ args ~ _) => FuncExpr(name, args, (aggregationFunctions contains name))
+  def cexpr =
+    ( expr ~ compareOperator ~ expr ^^ { case (lhs ~ op ~ rhs) => BinaryOpExpr(lhs, op, rhs) }
+    | expr
+    )
+
+  def lexpr =
+    ( "if" ~> (cond ~ ("then" ~> expr) ~ rep(elseIfExprs) ~ opt("else" ~> expr)) <~ "end" ^^ {
```

Comment:
I'm afraid that's not the case for most keywords, e.g., `function`, `over`, `returns`, `handles tsv lines`, `@weight`, `@label`.  I'd say `NULL`, `TRUE`, `FALSE`, `EXISTS`, `ALL`, `OPTIONAL` are rather exceptions and the pretty printer even prints lower case for some of them (`true`, `false`).  I'd go for ignoring the case in a future update (I actually have a partial patch towards this but somehow encountered performance bugs while testing).